bin/commands/stop.py

```diff
@@ -0,0 +1,13 @@
+import utils
+
+DESCRIPTION = "Stop docker containers"
+
+
+def setup(parser):
+    parser.add_argument("image", metavar="IMAGE", type=str, nargs="?", help="Docker image")
+
+
+def run(args):
+    image_name = args.image or utils.get_running_image_name()
+    utils.logger.info("Stop container for image: {}".format(image_name))
+    utils.run_docker_compose(["stop"], image_name)
```

Comment:
Proposal:

```python
    utils.run_docker_compose(["down"], image_name)
```

This way we ensure the start is also executed by ``d2-docker`` as containers and networks are removed.

Note: By default volumes are persisted, so we wouldn't loose data by removing the containers.